content/building/environment-variable-groups.md

```diff
@@ -4,15 +4,16 @@ title: Environment variable groups
 weight: 3
 ---
 
-Environment variable groups allow you to define and store related values that are reusable throughout your [codemagic.yaml](../getting-started/yaml/) applications and workflows. A variable *group* tags a set variables that can be imported together in codemagic.yaml file. For example, you could define a `staging` group for variables related to your staging deployment and a `production` group for variables related to your production deployment. The variable names in staging and production groups can be identical, but the values will be set depending on which group is imported in the workflow. This allows you to reuse variable definitions in scripts but assign the actual values based on the group.
+Environment variable groups allow you to define and store related set of variables that are reusable in your [codemagic.yaml](../getting-started/yaml/) workflows. A variable *group* tags a set variables that can be imported together in a codemagic.yaml file. For example, you could define a `staging` group for variables related to your staging deployment and a `production` group for variables related to your production deployment. The variable names in staging and production groups can be identical, but the values will be set depending on which group is imported in the workflow. This allows you to reference variables in reusable scripts, but assign the actual values per workflow based on the imported group.
```

Comment:
'Environment variable groups allow you to define and store related setS of variables', maybe? 

and 'A variable *group* tags a set OF variables'

Reply:
Good catch. I'm starting to question my English skills. 😅